.gitattributes

```diff
@@ -9,6 +9,46 @@
 # force changelog merging to use union
 html/changelog.html merge=union
 
-# Declare files that will always have CRLF line endings on checkout.
-*.dm text eol=crlf
-*.dmm text eol=crlf
```

Comment:
The hell is wrong with you all?

Reply:
blame bay https://github.com/Baystation12/Baystation12/blob/dev/.gitattributes